cmake/versions.cmake

```diff
@@ -1,9 +1,10 @@
 # Version for libraries CPP
-SET(VERSION "2.11.99")
+SET(VERSION "2.10.99")
 STRING(REGEX MATCHALL "[0-9]+" VERSION_PARTS "${VERSION}")
 LIST(GET VERSION_PARTS 0 VERSION_MAJOR)
 LIST(GET VERSION_PARTS 1 VERSION_MINOR)
-SET(SOVERSION "${VERSION_MAJOR}.${VERSION_MINOR}")
+LIST(GET VERSION_PARTS 2 VERSION_PATCH)
+SET(SOVERSION "${VERSION_MAJOR}.${VERSION_MINOR}.${VERSION_PATCH}")
```

Comment:
@firewave Do you see why we don't just assign SOVERSION directly, either :

`SET(SOVERSION "2.10.99")`

or

```
SET(VERSION "2.10.99")
SET(SOVERSION VERSION)
```